scrolltoggle/index.js

```diff
@@ -18,11 +23,17 @@ var defaultConfig = {
  */
 function ScrollToggle(el, config) {
   this.el_ = el;
-  // TODO(stevenle): support config overrides per element using a data- attr.
-  this.config_ = config;
-  this.lastScrollPos_ = 0;
+
+  this.config_ = objects.clone(config);
+  if (this.el_.hasAttribute('data-scrolltoggle')) {
```

Comment:
OK, sounds good. I think we should be as consistent as possible. `ak-<module>` for CSS defaults and `data-ak-<module>` for data attributes SGTM.